modules/rest-api/pages/rest-configure-saslauthd.adoc

```diff
@@ -1,61 +1,85 @@
+
+== Responses
+
+Successful use of GET returns `200 OK`, and a JSON object containing three key-value pairs, which respectively indicate whether saslauthd is enabled, the current list of administrators, and the current list of read-only administrators.
+
+Successful use of POST returns `200 OK`, and a JSON object containing three key-value pairs, which respectively indicate whether saslauthd is enabled, the current list of administrators, and the current list of read-only administrators.
+
+A malformed URI returns `404 Object Not Found`.
+
+Failure to authenticate returns `401 Unauthorized`.
+Successful authentication with an inappropriate role returns `403 Forbidden`, and an object such as the following: `{"message":"Forbidden. User needs the following permissions","permissions":["cluster.admin.security.external!write"]}`.
+
+== Examples
+
+The following example assigns saslauthd roles to three users, who have already been established as members of the `sasl` group on a Ubuntu Linux system:
 
 ----
-GET /settings/saslauthdAuth
-Host: localhost:8091
-Authorization: Basic xxxxxxxxxxxxxxxxxxx
-Accept: */*
-Content-Type: application/x-www-form-urlencoded
+curl -X POST http://localhost:8091/settings/saslauthdAuth -u Administrator:password -d admins=[couchbase] -d roAdmins=[abc,def] -d enabled=true
 ----
 
-*Example*
+If the call is successful, an object such as the following is returned.
 
 ----
-curl -v -X GET -u Administrator:password \
-http://10.143.192.101:8091/settings/saslauthdAuth \
--d enabled=true \
--d admins=alice,barry \
--d roadmins=clair,daniel
+{"enabled":true,"admins":["[couchbase]"],"roAdmins":["[abc,def]"]}
 ----
 
-.Controller parameters
-[cols="40,70"]
-|===
-| Function | Description
+The following example retrieves the current saslauthd settings for the cluster:
 
-| admins
-| Comma-separated list of SASL users to be given administrator privileges on the cluster.
-If either an asterisk or the word 'asterisk' is specified as the value, all SASL users not specified to be granted a different set of privileges are granted administrator privileges.
+----
+curl -X GET http://localhost:8091/settings/saslauthdAuth -u Administrator:password
+----
+
+If the call is successful, an object such as the following is returned:
 
-| roadmins
-| Comma-separated list of SASL users to be given read-only administrator privileges on the cluster.
-If either an asterisk or the word 'asterisk' is specified as the value, all SASL users not specified to be granted a different set of privileges are granted read-only administrator privileges.
+----
+{"enabled":true,"admins":["[tjh]"],"roAdmins":["[tjh,ajh]"]}
+----
 
-| enabled
-| Enables or disables SASL on the cluster. 1 enables, 0 disables.
+== See Also
```

Comment:
Suggest adding an example of disabling and how it deletes the admins/roAdmins

$ curl -u Administrator:asdasd localhost:8091/settings/saslauthdAuth
{"enabled":true,"admins":["steve"],"roAdmins":["alycia"]}

$ curl -u Administrator:asdasd localhost:8091/settings/saslauthdAuth -d "enabled=false"
{"enabled":false,"admins":[],"roAdmins":[]}

Reply:
I've added.